src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -1347,29 +1308,32 @@ private IInvalidStatement CreateBoundBadStatementOperation(BoundBadStatement bou
             return new LazyInvalidStatement(children, _semanticModel, syntax, type, constantValue, isImplicit);
         }
 
-        private IVariableDeclaration CreateVariableDeclaration(BoundLocalDeclaration boundLocalDeclaration)
+        private IOperation CreateBoundLocalDeclarationOperation(BoundLocalDeclaration boundLocalDeclaration)
         {
-            // special case for variable decl due to BoundNode using same node for both statement and other construct
-            return _variableDeclarationCache.GetOrAdd(boundLocalDeclaration, d =>
+            if (boundLocalDeclaration.Syntax.Kind() == SyntaxKind.LocalDeclarationStatement)
             {
-                return OperationFactory.CreateVariableDeclaration(d.LocalSymbol, Create(d.InitializerOpt), _semanticModel, d.Syntax);
-            });
-        }
+                var statement = (LocalDeclarationStatementSyntax)boundLocalDeclaration.Syntax;
+                var variableDeclarator = statement.Declaration.Variables.First();
```

Comment:
use declarator for IVariableDeclaration since that is what multiple local declaration is using to make things consistent.